specs/swagger.yml

```diff
@@ -1096,21 +1339,39 @@ definitions:
         type: boolean
       bluetooth:
         type: boolean
+      hal:
+        type: boolean
+      mongo:
+        type: boolean
+      influx:
+        type: boolean
+      grafana:
+        type: boolean
+      isolatedDockerContainer:
+        type: boolean
       logFileCount:
         type: number
       version:
-        type: string      
+        type: string
+      isReadyToUpgrade:
+        type: boolean
+      isReadyToRollback:
+        type: boolean
       statusFrequency:
-        type: number      
-      configFrequency:
-        type: number      
+        type: number
+      changeFrequency:
+        type: number
+      scanFrequency:
+        type: number
+      proxy:
+        type: string
       typeKey:
```

Comment:
I think it's better to have `type` as a JSON property here.
```
type:
  type: object
  properties:
    key:
      type: number
    id:
      type: number
    name:
      type: string
    image:
      type: string
    description:
      type: string
```

Reply:
do you mean fogType? 
like
fogType:
  type: number